### PR TITLE
Fixed the bug uploading a tmx or a glossary

### PR DESCRIPTION
**Pull request by riccio82**

### CODE REVIEW

File: public/js/lib/fileupload/main.js [hunk 1]
```diff
@@ -202,24 +202,26 @@ UI = {
     },
 
     TMXloaded: function () {
-        console.log( 'ecco: ', $( '#private-tm-key' ).val() );
-        $( '#disable_tms_engine' ).trigger( 'click' );
+        // $( '#disable_tms_engine' ).trigger( 'click' );
```
Comment (freegenie) [inline public/js/lib/fileupload/main.js]: @riccio82 why keep this line around?

File: public/js/new-project.js [hunk 2]
```diff
@@ -303,6 +249,53 @@ $(document).ready(function() {
...
+
+	APP.createTMKey = function () {
+
+		if($(".mgmt-tm .new .privatekey .btn-ok").hasClass('disabled')) {
+			return false;
+		}
+
+
+		//call API
+		APP.doRequest( {
```
Comment (freegenie) [inline public/js/new-project.js]: let's always return `APP.doRequest` in such cases, so we can easily chain stuff if needed.

File: public/js/new-project.js [hunk 3]
```diff
@@ -352,27 +345,27 @@ $(document).ready(function() {
...
 			if(!$('#create_private_tm_btn[data-key]').length) {
 				$("input[id^='private-tm-']").prop("disabled", false);
 				$("#create_private_tm_btn").removeClass("disabled");
 			}
 		}
-	});
+	});*/
 
-	$("#private-tm-key").on('keyup', function(e) {
+	/*$("#private-tm-key").on('keyup', function(e) {
```
Comment (freegenie) [inline public/js/new-project.js]: you like keeping comments 😄

Comment (riccio82) [inline public/js/new-project.js]: :) I leave this comments because i dont know about this events, I added a card in trello to check this dead code.

**Pull request merged**